generators/heroku/index.js

```diff
@@ -31,31 +31,49 @@ module.exports = HerokuGenerator.extend({
         this.angularAppName = this.getAngularAppName();
         this.buildTool = this.config.get('buildTool');
         this.applicationType = this.config.get('applicationType');
+        this.herokuAppName = this.config.get('herokuAppName');
     },
 
     prompting: function () {
         var done = this.async();
 
-        var prompts = [
-            {
-                type: 'input',
-                name: 'herokuDeployedName',
-                message: 'Name to deploy as:',
-                default: this.baseName
-            },
-            {
-                type: 'list',
-                name: 'herokuRegion',
-                message: 'On which region do you want to deploy ?',
-                choices: ['us', 'eu'],
-                default: 0
-            }];
-
-        this.prompt(prompts).then(function (props) {
-            this.herokuDeployedName = _.kebabCase(props.herokuDeployedName);
-            this.herokuRegion = props.herokuRegion;
-            done();
-        }.bind(this));
+        if (this.herokuAppName && this.herokuAppName.length != 0) {
+            exec('heroku apps:info --json', function (err, stdout) {
+                if (err) {
+                    this.abort = true;
+                    this.log.error(err);
+                } else {
+                    var json = JSON.parse(stdout);
+                    this.herokuAppName = json['app']['name'];
```

Comment:
It indeed makes sense. But I have one more doubt. When you do `heroku apps:info --json` how do we know we are taking the correct app name if we have more than one app?  I have not used heroku much but I assume its a possibility rite?